src/generic/grid.cpp

```diff
@@ -249,6 +249,20 @@ int wxGridColumnOperations::GetFirstLine(const wxGrid *grid, wxGridWindow *gridW
     return grid->GetNumberFrozenCols();
 }
 
+void wxGridOperations::PrepareDCForLabels(wxGrid *grid, wxDC &dc) const
+{
```

Comment:
I think there should be a comment in this function explaining what it does and why, as it's not really obvious.

I'd also like to find some less ugly way of doing it, but I don't really see any right now... The only nice solution would be to factor out some `PrepareDCForScrollPos(wxDC& dc, int x, int y)` from `wxScrollHelperBase::DoPrepareDC()` and then call it with one of `x` or `y` set to the actual scroll position and the other one set to 0, but I'm not 100% sure it's worth it. OTOH it would definitely make this function short and simple enough to not need any comment!